packages/plugin-typescript/README.md

```diff
@@ -0,0 +1,167 @@
+# @code-pushup/typescript-plugin
+
+[![npm](https://img.shields.io/npm/v/%40code-pushup%2Ftypescript-plugin.svg)](https://www.npmjs.com/package/@code-pushup/typescript-plugin)
+[![downloads](https://img.shields.io/npm/dm/%40code-pushup%2Ftypescript-plugin)](https://npmtrends.com/@code-pushup/typescript-plugin)
+[![dependencies](https://img.shields.io/librariesio/release/npm/%40code-pushup/typescript-plugin)](https://www.npmjs.com/package/@code-pushup/typescript-plugin?activeTab=dependencies)
+
+🕵️ **Code PushUp plugin for measuring TypeScript quality with compiler diagnostics.** 🔥
+
+This plugin allows you to **incrementally adopting strict compilation flags in TypeScript projects**.
+It analyzes your codebase using the TypeScript compiler to detect potential issues and configuration problems.
+
+TypeScript compiler diagnostics are mapped to Code PushUp audits in the following way:
+
+- `value`: The number of issues found for a specific TypeScript configuration option -> 3
+- `displayValue`: The number of issues found -> 3 issues
```

Comment:
It doesn't seem like the display value formatting is actually implemented, BTW :confused: 

![image](https://github.com/user-attachments/assets/9df11c9f-3293-46b9-b593-0f4edf22eb71)